tests/pos/convert.scala

```diff
@@ -1,7 +1,5 @@
-import Conversion.convert
-
 given Conversion[String, Int] = _.length
 given Conversion[Int, String] = _.toString
 
 def f(x: String): Int = x.convert
-def g(x: Int): String = x.convert[String]
+def g(x: Int): String = x.convert
```

Comment:
This won't work if there are two `Conversion[Int, Y]` in scope for two different `Y`s.

Reply:
It does with a suitable type ascription, see updated test.

Reply:
@sjrd Looks like @rjolly is right. Do you agree?

Reply:
Yes, apparently it works, indeed.